src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/BaseOtlpExportClient.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="BaseOtlpExportClient.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.ExportClient
+{
+    /// <summary>Base class for sending OTLP export requests.</summary>
+    internal abstract class BaseOtlpExportClient
+    {
+        protected BaseOtlpExportClient(OtlpExporterOptions options)
+        {
+            Guard.Null(options, nameof(options));
+            Guard.InvalidTimeout(options.TimeoutMilliseconds, nameof(options.TimeoutMilliseconds));
+
+            this.Options = options;
+
+#if NETCOREAPP3_1
+            EnsureUnencryptedSupportIsEnabled(options);
+#endif
+        }
+
+        internal OtlpExporterOptions Options { get; }
+
+        private static void EnsureUnencryptedSupportIsEnabled(OtlpExporterOptions options)
```

Comment:
Is it also needed for `BaseOtlpHttpExportClient`? MS doc only says about gRPC.

Reply:
As far as I know, it's for both:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTests.cs#L35-L36

Reply:
This test uses gRPC protocol. Can you double-check it?

Reply:
But the docs say:
> If your application is targeting .NET Core 3.1, and you are using an insecure (HTTP) endpoint, the following switch must be set before adding OtlpExporter.

So that means both scenarios?

Reply:
(from what I've seen, this is required for both scenarios)

Reply:
I will double-check it then 😉 ETA 1 hour

Reply:
It is indeed needed. I am creating a PR to add `HttpProtobuf` to the integration test 😉

Reply:
Happy to run the tests against both protocols if you want?

Reply:
It occurs that I have a problem making the HTTP protocol to work.
See: https://github.com/open-telemetry/opentelemetry-dotnet/pull/2692